### ISSUE TITLE
Sub categories issues in forums

### ISSUE THREAD

**Issue by Ylmzef** [moxi9/phpfox#240]
Sub categories not showing or showing worng under forum top categories page.
Like this: http://community.phpfox.com/forum/70/community-support/ "Nebula - v3" not listed.
http://community.phpfox.com/forum/44/legacy-versions/ showing some wrong sub categories.

**Comment by Data66PF** [comment]
duplicate #21 I think

**Comment by Ylmzef** [comment]
Still issue!
http://community.phpfox.com/forum/120/phpfox-announcements/
This showing 20 THREADS but there listed 18 thereads..
http://community.phpfox.com/forum/121/announcement-discussions/

http://community.phpfox.com/forum/
Under Archive: Discussions showing 6 categories but here listed 8 categories.
http://community.phpfox.com/forum/1/archive-discussions/
Site Showcase and Improve your phpFox site not showing on forums page.

**Comment by Data66PF** [comment]
We've not updated the counters. All that is needed to be done is the counters updated, run that tool. I'll ask if I can run them Monday.

**Comment by Ylmzef** [comment]
Please update the ticket when you done.

**Comment by Ylmzef** [comment]
Still waiting for this..

**Comment by Data66PF** [comment]
Confirmed on my 4.0.9 dev site. I have a child forum in the Discussions > General forum. If I click the main Discussions category, it should list General, Movies, Music, ChildinCat (one made for the tests).

If I then were to click the General category, it would then show childsub (the one I made for this). 

However it doesn't show that. It shows this when I click the main Discussions category:

childsub, Movies, Music, ChildinCat  

Notice above that it didn't show the parent General but instead showed its child forum called childsub.
